README.md

```diff
@@ -4,52 +4,65 @@
 - [Commits & releases](#commits--releases)
-- [Testing changes locally](#testing-changes-locally)
+- [Testing with your own project locally](#testing-with-your-own-project-locally)
 - [Get involved](#get-involved)
 - [Reach out to us](#reach-out-to-us)
   - [You found a bug or want to propose a feature?](#you-found-a-bug-or-want-to-propose-a-feature)
 - [License](#license)
 - [Code of Conduct](#code-of-conduct)
 - [Contributors ✨](#contributors-)
-  <!-- /TOC -->
 
-## Packages
+<!-- /TOC -->
 
-Forma 36 is a monorepo maintained using Lerna. Get started with Lerna by following this link: https://github.com/lerna/lerna
+## Packages
 
-The monorepo is currently structured into the following packages:
+Forma 36 is a monorepo structured into different packages and maintained using [Lerna](https://github.com/lerna/lerna).
+The packages available are:
 
 - [`forma-36-website`](https://github.com/contentful/forma-36/tree/master/packages/forma-36-website)
-  - This package includes the design system documentation which offers guidelines, best practices and examples.
-    See the README [here](./packages/forma-36-website/README.md)
+  This package includes the design system documentation which offers guidelines, best practices and examples.
+
 - [`forma-36-react-components`](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components)
-  - A React component library built with Storybook. See the README [here](./packages/forma-36-react-components/README.md)
+  A React component library built with Storybook.
+
 - [`forma-36-fcss`](https://github.com/contentful/forma-36/tree/master/packages/forma-36-fcss)
-  - CSS utility classes for spacing, typography, colors and transitions. See the README [here](./packages/forma-36-fcss/README.md)
+  CSS utility classes for spacing, typography, colors and transitions.
+
 - [`forma-36-tokens`](https://github.com/contentful/forma-36/tree/master/packages/forma-36-tokens)
-  - Forma 36 design tokens. See the README [here](./packages/forma-36-tokens/README.md)
+  This package includes all the visual tokens (e.g. colors, margins, font sizes, etc) that used in the react library
 
-### Adding packages
+### Creating new packages
 
-You can use [Plop](https://plopjs.com/) to scaffold new component package. Run `yarn generate` and follow the steps in the CLI to create a component package. Using your input here, Plop will generate the relevant files and add the relevant imports/exports to the main `src/index.ts` file required to make the component available when publishing the library.
+We use use [Plop](https://plopjs.com/) create scripts that help you to scaffold new packages. On the root of the repo, you can run `yarn generate`. Then follow the steps in the CLI. Plop will generate the relevant files and add the relevant imports and exports to the main `src/index.ts` file required to make the component available when publishing the library.
```

Comment:
"We use Plop create" -> "We use Plop *to* create".

I'd also say "On the root of the repo" should use "In" or "At the root".